pandas/io/clipboard/__init__.py

```diff
@@ -68,8 +69,8 @@ def determine_clipboard():
             return init_gtk_clipboard()
 
         try:
-            # Check if PyQt4 is installed
-            import PyQt4  # noqa
+            # Check if qtpy is installed
+            import qtpy  # noqa
```

Comment:
Shouldn't this fall back to `PyQt4` if `qtpy` isn't available? I'm Ok with making `qtpy` the default, but we don't want to break a user's install that just has PyQt4.